gui-wm/wayfire/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <longdescription>
```

Comment:
I usually put this after the maintainers, but not sure if necessary.

Reply:
I've seen both. I think it is fine :confused: